test/src/test/java/org/corfudb/integration/CorfuReplicationLocalClusterMissingIT.java

```diff
@@ -0,0 +1,174 @@
+            streamName,
+            Sample.StringKey.class,
+            Sample.IntValue.class,
+            Sample.Metadata.class,
+            TableOptions.builder().schemaOptions(
+                    CorfuOptions.SchemaOptions.newBuilder()
+                        .setIsFederated(true)
+                        .addStreamTag(ObjectsView.LOG_REPLICATOR_STREAM_INFO.getTagName())
+                        .build())
+                .build()
+        );
+
+        assertThat(mapSource.count()).isZero();
+        assertThat(mapSink.count()).isZero();
+
+        configTable = sourceCorfuStore.openTable(
+            DefaultClusterManager.CONFIG_NAMESPACE, DefaultClusterManager.CONFIG_TABLE_NAME,
+            ClusterUuidMsg.class, ClusterUuidMsg.class, ClusterUuidMsg.class,
+            TableOptions.fromProtoSchema(ClusterUuidMsg.class)
+        );
+
+        try (TxnContext txn = sourceCorfuStore.txn(DefaultClusterManager.CONFIG_NAMESPACE)) {
+            txn.putRecord(configTable, topologyType, topologyType, topologyType);
+            txn.commit();
+        }
+
+        assertThat(configTable.count()).isOne();
+    }
+
+    @Test
+    public void testLocalClusterAbsent() throws Exception {
+        // Write 10 entries to source map
+        for (int i = 0; i < 10; i++) {
+            try (TxnContext txn = sourceCorfuStore.txn(NAMESPACE)) {
+                txn.putRecord(mapSource, Sample.StringKey.newBuilder().setKey(String.valueOf(i)).build(),
+                    Sample.IntValue.newBuilder().setValue(i).build(), null);
+                txn.commit();
+            }
+        }
+        assertThat(mapSource.count()).isEqualTo(10);
+        assertThat(mapSink.count()).isZero();
+
+        int firstBatch = 10;
+        log.info("Before log replication, append {} entries to source map. Current source corfu" +
+                "[{}] log tail is {}, sink corfu[{}] log tail is {}", firstBatch, sourceClusterCorfuPort,
+            sourceRuntime.getAddressSpaceView().getLogTail(), sinkClusterCorfuPort,
+            sinkRuntime.getAddressSpaceView().getLogTail());
+
+        sinkReplicationServer = runReplicationServer(sinkReplicationServerPort, sinkClusterCorfuPort,
+            pluginConfigPath, transportType);
+
+        sourceReplicationServer = runReplicationServer(sourceReplicationServerPort, sourceClusterCorfuPort,
+            pluginConfigPath, transportType);
+
+        log.info("Replication servers started");
+        Thread.sleep(10000);
+
+        assertThat(mapSink.count()).isZero();
+
+        try (TxnContext txn = sourceCorfuStore.txn(DefaultClusterManager.CONFIG_NAMESPACE)) {
+            txn.putRecord(configTable, DefaultClusterManager.OP_LOCAL_CLUSTER_NOT_FOUND,
+                DefaultClusterManager.OP_LOCAL_CLUSTER_NOT_FOUND,
+                DefaultClusterManager.OP_LOCAL_CLUSTER_NOT_FOUND);
+            txn.commit();
+        }
+        assertThat(configTable.count()).isOne();
+        //log.info("New topology config applied!");
```

Comment:
nit: we can remove this

Reply:
Cleanup for the tests is in progress.  Will post another commit shortly.